src/content/docs/en/guides/cms/decap-cms.mdx

```diff
@@ -8,13 +8,92 @@ i18nReady: true
 ---
 import Grid from '~/components/FluidGrid.astro'
 import Card from '~/components/ShowcaseCard.astro'
+import FileTree from "~/components/FileTree.astro"
 
 [Decap CMS](https://www.decapcms.org/) (formerly Netlify CMS) is an open-source, Git-based content management system.
+
+Decap allows you to take full advantage of all of Astro's features, including image optimization and content collections.
+
+Decap adds a route (typically `/admin`) to your project that will load a React app to allow authorized users to manage content directly from the deployed website. Decap will commit changes to your Astro project's source repository.
+
+## Installing DecapCMS
+
+There are two options for adding Decap to Astro:
+
+1. [Install Decap via a package manager](https://decapcms.org/docs/install-decap-cms/#installing-with-npm) with the following command:
+
+	```sh
+	npm install decap-cms-app --save
+	```
+ 
+1. Import the package into a `<script>` tag in your page `<body>`
+
+	```html title='/admin'
+    <body>
+      <!-- Include the script that builds the page and powers Decap CMS -->
+      <script src="https://unpkg.com/decap-cms@^3.1.1/dist/decap-cms.js"></script>
+    </body>
+	```
+ 
+
+## Configuration
+
+1. Create a static admin folder at `public/admin/`
+2. Add `config.yml` to `public/admin/`: 
+  <FileTree>
+  - admin
+    - index.html
+    - config.yml
+  </FileTree>
+3. Configure your collections schemas in `config.yml`
+  ```yml
+  collections:
+    - name: "blog" # Used in routes, e.g., /admin/collections/blog
+      label: "Blog" # Used in the UI
+      folder: "src/content/blog" # The path to the folder where the documents are stored
+      create: true # Allow users to create new documents in this collection
+      fields: # The fields for each document, usually in front matter
+        - { label: "Layout", name: "layout", widget: "hidden", default: "blog" }
+        - { label: "Title", name: "title", widget: "string" }
+        - { label: "Publish Date", name: "date", widget: "datetime" }
+        - { label: "Featured Image", name: "thumbnail", widget: "image" }
+        - { label: "Rating (scale of 1-5)", name: "rating", widget: "number" }
+        - { label: "Body", name: "body", widget: "markdown" }
+  ```
+5. (optional) To enable media uploads to a specific folder via the Decap editor, add an appropriate path:
```

Comment:
```suggestion

4. Add the `admin` route for your React app. This file can be either `public/admin/index.html` alongside your `config.yml` or, if you prefer to use an Astro route, `src/pages/admin.astro`.

		<FileTree>
				- public
			    - admin
			    	- config.yml
			    	- index.html
		</FileTree>

5. (optional) To enable media uploads to a specific folder via the Decap editor, add an appropriate path:
```
Is there a sample of what this file (html or astro) should look like? Can it just be a blank file, or do you need to put something in here? I would either show an example (if it can't be blank) or link to DecapCMS docs to an example of what this file looks like.